TweakAccessorGenerator/TweakAccessorGenerator/Extensions/NSNumber+ValueTypes.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NSNumber+ValueTypes.swift
+//  Copyright © 2021 Just Eat Takeaway. All rights reserved.
+//
+
+import Foundation
+
+public extension NSNumber {
+    
+    var tweakType: String {
+        let encoding = String(cString: self.objCType)
+        switch encoding {
+        case "d":
+            return "Double"
+                
+        case "q":
+            return "Int"
+            
+        case "c":
+            return "Bool"
+            
```

Comment:
This method is not comprehensive as `q` will apply for a signed long long, not all forms of int within an NSNumber. `c` could also be any signed char, not just a boolean. 
Are you happy with this?

Reply:
Yes, I'm happy with this, it's annoying in the first place we have to do this. We had this already in JustTweak, in fact it was copied from [here](https://github.com/justeat/JustTweak/blob/master/JustTweak/Classes/Extensions/TweakExtensions.swift#L23). Those cases and implementation should be enough, I think we have enough tests to give us confidence.